src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,117 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+    static RegistrationService registrationService;
+    static User normalUser1;
+    static User normalUser2;
+    static User testUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        normalUser1 = new User();
+        normalUser1.setLogin("Vasya");
+        normalUser1.setAge(19);
+        normalUser1.setPassword("abracadabra");
+        normalUser2 = new User();
+        normalUser2.setLogin("Kate");
+        normalUser2.setAge(33);
+        normalUser2.setPassword("MAGA2024");
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+      }
+
+    @Test
+    void register_nullAge_notOk() {
+        testUser = new User();
```

Comment:
we can move testUser = new User(); to BeforeAll, and dont initialize it each test

Reply:
Can screw up like this.
For instance, if we assign testUser = null, subsequent tests will throw NPE
Can move it to BeforeEach, though)